admin-dev/themes/new-theme/template/header.tpl

```diff
@@ -45,37 +45,37 @@
     var youEditFieldFor = '';
     {if isset($shop_context)}
     {if $shop_context == Shop::CONTEXT_ALL}
-    youEditFieldFor = '{l s='This field will be modified for all your shops.' js=1}';
+    youEditFieldFor = '{l s='This field will be modified for all your shops.' js=1 d='Admin.Notifications.Info'}';
     {elseif $shop_context == Shop::CONTEXT_GROUP}
-    youEditFieldFor = '{l s='This field will be modified for all shops in this shop group:' js=1} <b>{$shop_name|@addcslashes:'\''}</b>';
+    youEditFieldFor = '{l s='This field will be modified for all shops in this shop group:' js=1 d='Admin.Notifications.Info'} <b>{$shop_name|@addcslashes:'\''}</b>';
     {else}
-    youEditFieldFor = '{l s='This field will be modified for this shop:' js=1} <b>{$shop_name|@addcslashes:'\''}</b>';
+    youEditFieldFor = '{l s='This field will be modified for this shop:' js=1 d='Admin.Notifications.Info'} <b>{$shop_name|@addcslashes:'\''}</b>';
     {/if}
     {/if}
-    var new_order_msg = '{l s='A new order has been placed on your shop.' js=1}';
-    var order_number_msg = '{l s='Order number:' js=1} ';
-    var total_msg = '{l s='Total:' js=1} ';
-    var from_msg = '{l s='From:' js=1} ';
-    var see_order_msg = '{l s='View this order' js=1}';
-    var new_customer_msg = '{l s='A new customer registered on your shop.' js=1}';
-    var customer_name_msg = '{l s='Customer name:' js=1} ';
-    var new_msg = '{l s='A new message was posted on your shop.' js=1}';
-    var see_msg = '{l s='Read this message' js=1}';
+    var new_order_msg = '{l s='A new order has been placed on your shop.' js=1 d='Admin.Navigation.Header'}';
+    var order_number_msg = '{l s='Order number:' js=1 d='Admin.Navigation.Header'} ';
+    var total_msg = '{l s='Total:' js=1 d='Admin.Global'} ';
+    var from_msg = '{l s='From:' js=1 d='Admin.Global'} ';
+    var see_order_msg = '{l s='View this order' js=1 d='Admin.Orderscustomers.Feature'}';
+    var new_customer_msg = '{l s='A new customer registered on your shop.' js=1 d='Admin.Navigation.Header'}';
+    var customer_name_msg = '{l s='Customer name:' js=1 d='Admin.Navigation.Notification'} ';
```

Comment:
Same than for the previous occurrence: I would localize it either in _Admin.Navigation.Header_ or _Admin.Global_, it is more relevant to me.